dist/theme/AlternativeLight.colortheme

```diff
@@ -0,0 +1,21 @@
+[Info]
+Name=Alternative (Light)
+Description=Alternative color theme. Light flavour.
+
```

Comment:
@evsh:  This is specifically a Light theme.  Please add `Inherits=Default (Light)`.